src/hal/src/Kconfig

```diff
@@ -21,21 +21,18 @@ config SYSLINK_RX_DMA
         up recources for other things. DMA is a shared resource though
         and might conflict with other functionality in the future.
 
-config CRTP_OVER_UART2
-    bool "Move CRTP communication to UART2 instead of over radio"
-    default n
-    help
-        The CRTP (Crazy Real Time Protocol) communication is normally
-        sent over the radio link (via Syslink). By enabling this the
-        communication will instead be sent over UART2 to communicate
-        with e.g. a companion computer instead.
+config ENABLE_CPX
+  bool "Enable CPX on UART2"
```

Comment:
Does this only CPX for UART2? or is the entire stack enabled? Perhaps this should be a bit clearer

Reply:
This has been split up on two configs, but one forces the other so it's the same behavior as before but more clear.